core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -279,6 +279,7 @@ object RequestChannel extends Logging {
           .append(",principal:").append(session.principal)
           .append(",listener:").append(context.listenerName.value)
           .append(",clientInformation:").append(context.clientInformation)
+          .append(",isForwarded:").append(envelope.isDefined)
```

Comment:
Not 100% sure about our metrics conventions, but I think this should be `forwarded`